app/src/pages/embedding/ClusterSortPicker.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useMemo, useState } from "react";
+import { css } from "@emotion/react";
+
+import {
+  ActionButton,
+  DropdownMenu,
+  DropdownTrigger,
+  Item,
+  ListBox,
+} from "@arizeai/components";
+
+import { useDatasets, usePointCloudContext } from "@phoenix/contexts";
+import { ClusterSort } from "@phoenix/store";
+
+function getSortKey(sort: ClusterSort): string {
+  return `${sort.column}:${sort.dir}`;
+}
+export function ClusterSortPicker() {
+  const { referenceDataset } = useDatasets();
+  const hasReferenceDataset = !!referenceDataset;
+  const sort = usePointCloudContext((state) => state.clusterSort);
+  const setSort = usePointCloudContext((state) => state.setClusterSort);
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const items = useMemo(() => {
+    const dynamicItems = [];
+    if (hasReferenceDataset) {
+      dynamicItems.push({
+        label: "Most drift",
+        value: getSortKey({
+          column: "driftRatio",
+          dir: "desc",
+        }),
+      });
+      return [
+        ...dynamicItems,
+        {
+          label: "Largest",
```

Comment:
at first i had thought it was sorting metric value since the dropdown is directly above the metric values
```suggestion
          label: "Largest Cluster",
```